lib/oli/accounts.ex

```diff
@@ -597,4 +601,70 @@ defmodule Oli.Accounts do
       )
     )
   end
+
+  @doc """
+  Get all the available publications and products based on:
+    - User's linked author
+    - User's institution
+    - User's permission to access global content
+
+  ## Examples
+
+      iex> list_available_publications_and_products(nil, nil, false)
+      []
+
+      iex> list_available_publications_and_products(123, 1, true)
+      [{%Publication{project: %Project{}}, %Section{}}, ...]
+  """
+  def list_available_publications_and_products(nil, nil, false), do: []
```

Comment:
Adding this functionality to the `Oli.Accounts` context introduces a dependency cycle between `Oli.Accounts` and `Oli.Publishing` and`Oli.Delivery.Sections`.  I think these new functions should simply go into `Oli.Publishing'